package.json

```diff
@@ -12,17 +12,17 @@
     "ipfs": false
   },
   "scripts": {
-    "test": "node --max-old-space-size=8192 $(npm bin)/aegir test",
+    "test": "node --max-old-space-size=8192 node_modules/.bin/aegir test",
     "test:node": "aegir test -t node",
-    "test:browser": "node --max-old-space-size=8192 $(npm bin)/aegir test -t browser",
-    "test:webworker": "node --max-old-space-size=8192 $(npm bin)/aegir test -t webworker",
+    "test:browser": "node --max-old-space-size=8192 node_modules/.bin/aegir test -t browser",
+    "test:webworker": "node --max-old-space-size=8192 node_modules/.bin/aegir test -t webworker",
     "lint": "aegir lint",
     "build": "aegir build",
-    "release": "node --max-old-space-size=8192 $(npm bin)/aegir release ",
-    "release-minor": "node --max-old-space-size=8192 $(npm bin)/aegir release --type minor ",
-    "release-major": "node --max-old-space-size=8192 $(npm bin)/aegir release --type major ",
-    "coverage": "node --max-old-space-size=8192 $(npm bin)/aegir coverage --timeout 100000",
-    "coverage-publish": "anode --max-old-space-size=8192 $(npm bin)/aegir coverage --provider coveralls --timeout 100000"
+    "release": "node --max-old-space-size=8192 node_modules/.bin/aegir release ",
+    "release-minor": "node --max-old-space-size=8192 node_modules/.bin/aegir release --type minor ",
+    "release-major": "node --max-old-space-size=8192 node_modules/.bin/aegir release --type major ",
+    "coverage": "node --max-old-space-size=8192 node_modules/.bin/aegir coverage --timeout 100000",
+    "coverage-publish": "anode --max-old-space-size=8192 node_modules/.bin/aegir coverage --provider coveralls --timeout 100000"
```

Comment:
@dryajov why this change?

Reply:
`$()` breaks windows as it is a unix shell (bash?) think.

Reply:
Thanks!